pylint/constants.py

```diff
@@ -2,6 +2,11 @@
 # For details: https://github.com/PyCQA/pylint/blob/master/COPYING
 
 import re
+import sys
+
+from astroid.__pkginfo__ import version as astroid_version
```

Comment:
Can we grab it directly from `__pkginfo__` as that is the source of truth for the version?

Reply:
Putting this version in __pgfinfo__ creates a 10+ links circular import error. I originally put it there :(

Reply:
We can't because in `__pkginfo__` we're defining the dependencies for the setup.py so astroid is not installed yet. We could do it if we put the dependencies in the setup.py/pyproject.toml directly